### PR TITLE
Todos from migrations

### PR DESCRIPTION
**Pull request by thorrester**
### Summary
Various todo's that came up from migrations:

- Add better `delete_card` logic
   - Delete and verify sql record has been deleted before deleting storage object
- Add `log_parameters` to `ActiveRun`
- Inject repo and contact to project if provided
- Expand limit when listing projects
- Still testing different cleanup logic for ubunut workers

### CODE REVIEW

File: .github/workflows/lint-unit-tests.yml [hunk 1]
```diff
@@ -33,16 +33,44 @@ jobs:
       - run : make setup.project
       - run : make lints.ci
 
-  Unit-Test:
+  Unit-Test-Ubuntu:
+    needs: Lints
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.11"]
```
Comment (damonallison) [inline .github/workflows/lint-unit-tests.yml:40]: Add `3.10`?

Comment (damonallison) [reply .github/workflows/lint-unit-tests.yml:40]: Nevermind, looks like it's in `coverage`

**Review by damonallison — APPROVED** [review]
🔥

File: .github/workflows/lint-unit-tests.yml (hunk 1 above)
Comment (thorrester) [reply .github/workflows/lint-unit-tests.yml:40]: Yea, separated it out so we don't duplicate the test

**Pull request merged**